test/views.ts

```diff
@@ -35,6 +35,28 @@ export const views = {
         token: "SkillTemplateToken",
         type: "Alexa.Presentation.APL.ExecuteCommands",
       },
+      APLTCommand: {
```

Comment:
let's take all that's not related to session entities into a separate PR

Reply:
Updated.